scripts/c-to-cpp-includes.pl

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/perl
```

Comment:
Please re-work this as a script in `scripts/maintenance/` to be run by the maintenance automation.
Taking the name+path of the file being processed as `$1` (to detect which ones to skip), its content on `stdin` and send corrected (or not) output to `stdout`.

Reply:
See other comment about expecting this to be a one-shot. Especially if any discussion is going to happen about the script itself.

Reply:
IMO this should not be a one-off. Maintaining these changes is extra work for reviewers. Which makes no sense when we already have this script that can lift the workload "good enough".